CRToast/CRToastConfig.m

```diff
@@ -750,7 +762,8 @@ BOOL CRToastAnimationDirectionIsHorizontal(CRToastAnimationDirection animationDi
     return !CRToastAnimationDirectionIsVertical(animationDirection);
 }
 
-static CGFloat kCRCollisionTweak = 0.5;
+//Este valor estaba a 0.5. Lo cambiamos a variable de configuración porque a partir de iOS>=9 hace que detecte la colisión mal y es necesario ponerlo a 0.0 ahora ver kCRToastCollisionTweakKey
+//static CGFloat kCRCollisionTweak = 0.5;
```

Comment:
This could probably be deleted since it was added [here](https://github.com/cruffenach/CRToast/pull/231/files#diff-fbdf4a52b720b431318cc66bfa5499c4R293)